src/views/mfa-verify/PushForm.js

```diff
@@ -49,6 +52,20 @@ define(['okta'], function (Okta) {
         factorName: this.model.get('factorLabel'),
         deviceName: this.model.get('deviceName')
       });
+
+      // trigger push once DOM is fully loaded
+      if (this.isAutoPushEnabled()) {
+        _.defer(_.bind(this.submit, this));
+      };
+    },
+    preRender: function () {
+      if (this.isAutoPushEnabled()) {
```

Comment:
Let's just move this to initialize - i.e. where we're already doing the "if(this.isAutoPushEnabled)" check (L57). And since we're only doing the check in one place, we can probably also get rid of using an "isAutoPushEnabled" helper function.